src/test/java/green/projectile/ProjectileTest.java

```diff
@@ -0,0 +1,46 @@
+package green.projectile;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ProjectileTest {
+    Projectile projectile = new Projectile(31, 65);
+    projectile.setSeconds(2.7);
```

Comment:
You need to put this in the test itself. You can't put code here, only create member variables.